lib/rubocop/config_loader.rb

```diff
@@ -196,14 +201,17 @@ def yaml_safe_load(yaml_code, filename)
         if defined?(SafeYAML) && SafeYAML.respond_to?(:load)
           SafeYAML.load(yaml_code, filename,
                         whitelisted_tags: %w[!ruby/regexp])
-        else
+        # Ruby 2.6+
```

Comment:
Didn't you say this would be ok on 2.5+?

Reply:
I just reverted the exact comment that was removed in #6766 
I only checked the `start_time` bit and saw that it is available since 2.5

Reply:
ok I checked and the kwargs used here where introduced in psych 3.1 (which is used since ruby 2.6)